ssl-config/src/main/java/com/palantir/remoting1/config/ssl/SslSocketFactories.java

```diff
@@ -74,6 +88,24 @@ public static SSLContext createSslContext(SslConfiguration config) {
     }
 
     /**
+     * Create an {@link SSLContext} initialized from the provided certificates.
+     *
+     * @param trustCertificatesByAlias
+     *        a map of X.509 certificate in PEM or DER format by the alias to load the certificate as.
+     */
+    public static SSLContext createSslContext(Map<String, PemX509Certificate> trustCertificatesByAlias) {
+        TrustManager[] trustManagers = createTrustManagers(trustCertificatesByAlias);
+
+        try {
+            SSLContext sslContext = SSLContext.getInstance("TLS");
+            sslContext.init(new KeyManager[]{}, trustManagers, null);
+            return sslContext;
+        } catch (GeneralSecurityException e) {
+            throw Throwables.propagate(e);
+        }
```

Comment:
guessing this block of code also repeated: `private static SSLContext createSslContextFromTrustManagers(TrustManager[] trustManagers)`?